funtofem/driver/funtofem_nlbgs_driver.py

```diff
@@ -289,6 +308,23 @@ def _solve_unsteady_forward(self, scenario, steps=None):
                 body.transfer_loads(scenario, time_index)
                 body.transfer_heat_flux(scenario, time_index)
 
+                if self._debug:
```

Comment:
See above, comm check needed here, I'm not going to repeat this comment for the other ones.